tests/README.md

```diff
@@ -0,0 +1,74 @@
+# Test documentation
+
+## List of scripted integration tests
+
+1. Up
+    * file: up.bash
```

Comment:
Codacy found an issue: [[list-item-content-indent] Don’t use mixed indentation for children, remove 1 space](https://app.codacy.com/gh/cnescatlab/docker-cat/pullRequest?prid=6196008)